app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self.connection = sqlite3.connect("cinema.db")
+        self.table_name = "actor"

```

Comment:
Does this DB contain only one author?

Reply:
What u mean?

Reply:
DB has the name `actor` which means, DB contains only one author